package.json

```
@@ -1,82 +1,14 @@
 {
-  "name": "web3-ui",
-  "license": "MIT",
-  "version": "1.0.0",
-  "private": false,
-  "engines": {
-    "node": ">=16.0.0",
-    "yarn": "^1.5"
-  },
-  "description": "React Typescript Library Boilerplate",
-  "author": "Dhaiwat Pandya",
-  "homepage": "https://github.com/Developer-DAO/web3-ui/",
-  "bugs": {
-    "url": "https://github.com/Developer-DAO/web3-ui/issues"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/Developer-DAO/web3-ui/"
-  },
+  "private": "true",
```

according to https://docs.npmjs.com/creating-a-package-json-file#required-name-and-version-fields, name and version are required fields in package.json files. maybe we should keep some stuff here still for whatever commands might be run from the root directory
```suggestion
  "name": "@web3-ui/root",
  "license": "MIT",  
  "version": "0.0.0",
  "private": "true",
  "engines": {
    "node": ">=16.0.0",
    "yarn": "^1.5"
  },
```

---

@etr2460 From reading the resource you shared, those are all required for a published package. Since the package is private it won't be published. Would you consider all of that still required?

This is not the final version for the package in any case, but just trying to understand.